README.md

```diff
@@ -169,7 +169,7 @@ class IntegrationUtilityTest extends TestCase
     #[Test]
     public function returnsCorrectAmountOfSimpleProductsInSampleDataDb()
     {
-        $container = DbContainerBuilder::mysql()
+        $connection = DbContainerBuilder::mysql()
```

Comment:
Can you please instead rename it to container usage on the line below? I think a container works better for this use case.